design/Soft_finality_guarantee.md

```diff
@@ -0,0 +1,161 @@
+# The soft-finality guarantee 
+
+The decentralised POBI protocol links the rollups to the L1 block used as proof, which means that any L1 reorg
+results in an implicit L2 reorg. The finality of an L2 rollup is fully linked to the finality of an L1 block.
+
+During bootstrapping, we want a central sequencer to guarantee finality in around ~1 second and for the bridge to be decentralised 
+and resistant to reorgs.
+
+Note: See bridge and finality designs for more details
+
+
+## Problem and Requirements
+
+The central sequencer can only guarantee a consistent ordering of the transactions it receives from the Obscuro users.
+It can't offer guarantees over the finality of cross-chain transactions. E.g., deposits to the bridge.
```

Comment:
Can't offer guarantees until the next rollup, which is a very long time (e.g. 6+ hours).